### ISSUE TITLE
Error use Command blaklist list 

### ISSUE THREAD

**Issue by MahdiTeaR** [ralphkb/sentinel-tickets#82]
![Screenshot 2024-07-17 110454](https://github.com/user-attachments/assets/5962c87f-89c7-4551-893d-cf712980332d)

**Comment by mrc4tt** [comment]
It works fine, I can't debug the issue.
![image](https://github.com/user-attachments/assets/3e64c6ef-6c51-4519-912a-6d806fd288f7)

You can try restarting the bot or re-run the slash command. :-)

**Comment by mrc4tt** [comment]
Seems the command blacklist **list** is bugged.

```
4|ticketbo | 
4|ticketbo | 
4|ticketbo | SyntaxError: Unexpected end of input
4|ticketbo |     at internalCompileFunction (node:internal/vm:128:18)
4|ticketbo |     at wrapSafe (node:internal/modules/cjs/loader:1279:20)
4|ticketbo |     at Module._compile (node:internal/modules/cjs/loader:1331:27)
4|ticketbo |     at Module._extensions..js (node:internal/modules/cjs/loader:1426:10)
4|ticketbo |     at Module.load (node:internal/modules/cjs/loader:1205:32)
4|ticketbo |     at Module._load (node:internal/modules/cjs/loader:1021:12)
4|ticketbo |     at Module.require (node:internal/modules/cjs/loader:1230:19)
4|ticketbo |     at Hook._require.Module.require (/usr/lib/node_modules/pm2/node_modules/require-in-the-middle/index.js:101:39)
4|ticketbo |     at require (node:internal/modules/helpers:179:18)
4|ticketbo |     at Object.<anonymous> (/home/fsh/sentinel-tickets/commands/ContextMenu/ticketAlert.js:12:25)
4|ticketbo | [InteractionCreate] blacklist command error: Error: Received one or more errors
4|ticketbo | [InteractionCreate] blacklist command error: Error: Received one or more errors
```

**Comment by ralphkb** [comment]
Hey, thank you for reporting this issue. It is now fixed in this commit: 3a87a77
